.travis.yml

```diff
@@ -11,10 +11,11 @@ install:
 - go get -u github.com/client9/misspell/cmd/misspell
 before_script:
 - $(exit $(go fmt ./... | wc -l))
-- go vet ./...
+- if [[ ! -f 'go_test.mod' ]]; then cp go.mod go_test.mod; cp go.sum go_test.sum; fi
+- go vet -modfile=go_test.mod ./...
 - find . -type f -name "*.go" | xargs misspell -error -locale US
 - staticcheck ./...
 script:
-- go test -i -race ./...
-- go test -v -run=TestNoRace -p=1 ./...
-- if [[ "$TRAVIS_GO_VERSION" =~ 1.16 ]]; then ./scripts/cov.sh TRAVIS; else go test -race -v -p=1 ./... --failfast; fi
+- go test -modfile=go_test.mod -i -race ./...
+- go test -modfile=go_test.mod -v -run=TestNoRace -p=1 ./...
+- if [[ "$TRAVIS_GO_VERSION" =~ 1.16 ]]; then ./scripts/cov.sh TRAVIS; else go test -modfile=go_test.mod -race -v -p=1 ./... --failfast; fi
```

Comment:
Question: we use `-modfile` here and line above, but ./scripts/cov.sh is not. Is that a problem?

Reply:
Yes those entries were missing `-modfile` so have updated now.